lambdas/issuecredential/src/test/java/uk/gov/di/ipv/cri/passport/issuecredential/IssueCredentialHandlerTest.java

```diff
@@ -155,14 +157,13 @@ void shouldReturnCredentialsOnSuccessfulDcsCredentialRequest()
         JsonNode claimsSet = objectMapper.readTree(signedJWT.getJWTClaimsSet().toString());
 
         assertEquals(200, response.getStatusCode());
-        assertEquals(7, claimsSet.get("claims").size());
+        assertEquals(7, claimsSet.size());
```

Comment:
It will be after I move the @Context and Type params inside the "vc" claim. But for now they are still at the top level. Didn't want to include that in this PR as it might have made it a bit too big but I will fix that bit after this goes through